repose-aggregator/tests/performance-tests/roles/gatling/files/gatling.conf

```diff
@@ -12,10 +12,9 @@ gatling {
     }
   }
   data {
-    writers = [console, file, graphite] # The list of DataWriters to which Gatling write simulation data (currently supported : console, file, graphite, jdbc)
+    writers = [console, file, graphite]          # The list of DataWriters to which Gatling write simulation data (currently supported : console, file, graphite, jdbc)
     graphite {
-      host = "influxdb.openrepose.org"  # The host where the Carbon server is located
-      port = 13002                      # The port to which the Carbon server listens to (2003 is default for plaintext, 2004 is default for pickle)
+      host = "internal.influxdb.openrepose.org"  # The host where the a Graphite listener is located
```

Comment:
Even better since ideally we'll create another internal network for... other... stuff.